src/extension/rpc.ts

```diff
@@ -119,7 +126,12 @@ export function createRpcHandler<Messages extends MessageCollection>(
           sourceId: id,
           result,
         });
-      } catch (error) {
+      } catch (e) {
+        const error =
```

Comment:
Noting here that @phryneas mentioned https://github.com/sindresorhus/serialize-error which could be much more useful since it maintains the stack trace and other properties.